File: Portal/Models/Dto/ClaimDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Portal.Models.Dto
{
    public class ClaimDto
    {
        public int Id { get; set; }
        public int RoleId { get; set; }
        public string ClaimType { get; set; }
    }
}

File: Portal/ClientApp/src/app/_autogenerated/claimDto.ts

//----------------------------------------------
//  DTO interface and model
//----------------------------------------------

export interface IClaimDto  {
    id: number;
    roleId: number;
    claimType: string;
}

export class ClaimDto  implements IClaimDto{ 
    public id: number;
    public roleId: number;
    public claimType: string;

    constructor(id : number = null,  roleId : number = null,  claimType : string = null ) {
        
        this.id = id;
        this.roleId = roleId;
        this.claimType = claimType;
    }
}



